### PR TITLE
[EP-401] Fix properties (phase one)

### PR DESCRIPTION
**Pull request by singhhari**
# 📲 What

Fixing properties after phase one of Segment QA. Some of these are smaller changes like changing the type whereas some are updates to where we get the values from.

# 🤔 Why

Updates after receiving QA feedback for phase one of our Segment implementation.

### CODE REVIEW

File: KsApi/models/PaymentType.swift [hunk 1]
```diff
@@ -15,4 +15,15 @@ public enum PaymentType: String, Decodable {
...
+
+  public var trackingString: String? {
+    switch self {
+    case .applePay:
+      return "apple_pay"
+    case .googlePay:
+      return nil
+    case .creditCard:
+      return "credit_card"
+    }
```
Comment (moderateepheezy) [inline KsApi/models/PaymentType.swift:27]: `trackingString` does not have test coverage.

File: Library/Tracking/KSRAnalytics.swift [hunk 2]
```diff
@@ -1535,27 +1537,27 @@ public final class KSRAnalytics {
   private var deviceOrientation: String {
     switch self.device.orientation {
     case .faceDown:
-      return "Face Down"
+      return "face_down"
     case .faceUp:
-      return "Face Up"
+      return "face_up"
```
Comment (moderateepheezy) [inline Library/Tracking/KSRAnalytics.swift:1542]: No test coverage for this.

File: Library/Tracking/KSRAnalytics.swift [hunk 3]
```diff
@@ -1535,27 +1537,27 @@ public final class KSRAnalytics {
...
     case .landscapeLeft:
-      return "Landscape Left"
+      return "landscape_left"
```
Comment (moderateepheezy) [inline Library/Tracking/KSRAnalytics.swift:1544]: Same as this.

File: Library/Tracking/KSRAnalytics.swift [hunk 4]
```diff
@@ -1535,27 +1537,27 @@ public final class KSRAnalytics {
...
     case .landscapeRight:
-      return "Landscape Right"
+      return "landscape_right"
     case .portrait:
-      return "Portrait"
+      return "portrait"
     case .portraitUpsideDown:
-      return "Portrait Upside Down"
+      return "portrait_upside_down"
```
Comment (moderateepheezy) [inline Library/Tracking/KSRAnalytics.swift:1550]: Same as this.

File: Library/Tracking/KSRAnalytics.swift [hunk 5]
```diff
@@ -1535,27 +1537,27 @@ public final class KSRAnalytics {
...
     case .unknown:
-      return "Unknown"
+      return "unknown"
```
Comment (moderateepheezy) [inline Library/Tracking/KSRAnalytics.swift:1552]: Same as this.

**Review by moderateepheezy — CHANGES_REQUESTED** [review]

**Review by tobitech — APPROVED** [review]

**Review by moderateepheezy — APPROVED** [review]

**Pull request merged**